### ISSUE TITLE
HeaderID/EDF_DataBlockID key from EDF are not accuratly parsed

### ISSUE THREAD

**Issue by vallsv** [silx-kit/fabio#195]
On fabio 0.6.0 the first parsed key always contains the starting `{` of the header block. Which is not expected.

Fortunately it is always a `system` key like `HeaderID`, `EDF_DataBlockID`, this could not affect too much the users.

Here we can see `{\n` is part of the first key.

```
In [2]: i = fabio.open("workspace/fabio.git/testimages/a0009.edf")

In [3]: i.header

{
  "{\nEDF_DataBlockID": "0.Image.Psd", 
  "EDF_BinarySize": "2097152", 
  "EDF_HeaderSize": "1024", 
  "ByteOrder": "LowByteFirst", 
  "DataType": "UnsignedShort", 
  "Dim_1": "1024", 
  "Dim_2": "1024", 
  "Image": "0", 
  "HeaderID": "EH:000000:000000:000000", 
  "Size": "2097152", 
  "filename": "a0009.edf"
}
```

From all the files from `testimages`:
```
In [25]: for name in f:
    with fabio.open(name) as i:
        for k in i.header.keys():
            if "{" in k:
                print([k])
   ....:                 
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\r\nEDF_DataBlockID']
[u'{\r\nEDF_DataBlockID']
[u'{\nHeaderID']
[u'{\r\nEDF_DataBlockID']
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\nHeaderID']
[u'{\nEDF_DataBlockID']
[u'{\nHeaderID']
[u'{\nEDF_DataBlockID']
[u'{\nEDF_DataBlockID']
[u'{\nEDF_DataBlockID']
[u'{\nEDF_DataBlockID']
[u'{\nHeaderID']

```

**Comment by vallsv** [comment]
If needed it can be patched on the user side using 2 lines (but the order of the keys is lost):
```
image = fabio.open("../data/edf/medipix.edf")
malformed_key, value = image.header.popitem(False)
image.header[malformed_key.split()[-1]] = value
```

**Comment by vallsv** [comment]
Closed by #196
